libnavigation-core/src/main/java/com/mapbox/navigation/core/lifecycle/CarAppLifecycleOwner.kt

```diff
@@ -127,13 +127,7 @@ internal class CarAppLifecycleOwner : LifecycleOwner {
             } else {
                 activitiesForegrounded--
                 logger.i(TAG, Message("app onActivityStopped"))
-                if (lifecycleCreated == 0 &&
-                    activitiesCreated == 0 &&
-                    activitiesForegrounded == 0
-                ) {
-                    check(activitiesCreated == 0 && activitiesForegrounded == 0) {
-                        "onActivityStopped when no activities exist"
```

Comment:
This was an old `check` that no longer makes sense. Also it was impossible considering the condition above (race condition is not possible either because each call happens on ui thread)